### PR TITLE
Abridged LT computed from single LT

### PR DESCRIPTION
**Pull request by sarahertog**
Hi Tim,
This lt_single2abridged() is the function requested by Patrick that computes an abridged life table that is coherent with a single year life table per the HLT protocol.  I've tried to use existing DemoTools functions to derive each of the LT columns.  Any feedback appreciated! Thanks! Sara

### CODE REVIEW

File: R/lt_single2abridged.R [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  AgeInt <- age2int(Age = Age5, OAvalue = 5)
+  N      <- length(Age5)
+  
+  # compute abridged lifetable columns
+  lx     <- lx[Age %in% Age5]     
+  nLx    <- single2abridged(nLx)  
+  ex     <- ex[Age %in% Age5]     
+  ndx    <- lt_id_l_d(lx)         
+  nqx    <- ndx / lx 
+  nAx    <- (nLx - (AgeInt * c(tail(lx,-1), NA))) / ndx
```
Comment (timriffe) [inline R/lt_single2abridged.R:45]: `c(tail(lx,-1), NA))` is an interesting construct hadn't seen `tail()` used like that, I would have done `c(lx[-1], NA)` or `shift.vector(1:10,-1,NA)`. All identical in the end.

File: R/lt_single2abridged.R [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#'   \item{Tx}{numeric. Lifetable total years left to live above age x.}
+#'   \item{ex}{numeric. Age-specific remaining life expectancy.}
+#' }
+#' @export
+#' 
+lt_single2abridged <- function(lx,
+                               nLx,
+                               ex,
+                               Age = 1:length(lx) - 1,
+                               ...) {
```
Comment (timriffe) [inline R/lt_single2abridged.R:32]: `...` isn't passed on, so no strictly necessary.

File: R/lt_single2abridged.R [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+#'   \item{ndx}{numeric. Lifetable deaths distribution.}
+#'   \item{nLx}{numeric. Lifetable exposure.}
+#'   \item{Sx}{numeric. Survivor ratios in uniform 5-year age groups.}
+#'   \item{Tx}{numeric. Lifetable total years left to live above age x.}
+#'   \item{ex}{numeric. Age-specific remaining life expectancy.}
+#' }
+#' @export
+#' 
+lt_single2abridged <- function(lx,
+                               nLx,
```
Comment (timriffe) [inline R/lt_single2abridged.R:29]: I think only two of these vectors are needed. By identity, we have `ex = Tx/lx`, and `Tx` gives us `Lx`. Any two of these should suffice. One supposes that the whole lifetable is on hand, though, so maybe no need to aim for maximum parsimony?

**Review by timriffe — APPROVED** [review]
This is very clean, you're on a roll! Thanks for taking care of this one. Merging in now. Will add an `@examples` section to the header soon.

**Comment by timriffe** [comment]
I made little changes, now pushed, thanks again

**Comment by sarahertog** [comment]
Thank you, Tim!

**Pull request merged**